src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/RuntimeBroker.cs

```diff
@@ -540,10 +540,10 @@ public bool IsBrokerInstalledAndInvokable(AuthorityType authorityType)
                 return false;
             }
 
-            // WAM does not work on pure ADFS environments
-            if (authorityType == AuthorityType.Adfs)
+            // WAM only works with AAD
+            if (authorityType != AuthorityType.Aad)
```

Comment:
thought we got it to work with CIAM, so we do not want to support that flow?

Reply:
WebView2 doesn't provide good SSO with Windows and doesn't satisfy the "device managed" CA policy. So we don't want to use WV2 in that context (AAD). Otherwise it's great. 

WAM doesn't support CIAM because CIAM has moved from login.microsoft.com to tenant.ciamlogin.com